content/posts/wasi-experimental-http.md

```diff
@@ -0,0 +1,146 @@
+---
+title:
+  "An experimental outbound HTTP library for the WebAssembly System Interface"
+description:
+  "Send HTTP requests from Rust and AssemblyScript Wasm modules running in
+  Wasmtime"
+date: 2021-03-08 00:00:00 +0000 UTC
+authorname: "Radu Matei"
+author: "@matei_radu"
+authorlink: "https://twitter.com/matei_radu"
+image: "images/twitter-card.png"
+tags: ["wasm", "wasi"]
+---
+
+Over the last year, our team has been experimenting with executing WebAssembly
+workloads on the server using the [WebAssembly System Interface][wasi] and
+[Wasmtime][wasmtime], and while more and more [proposals][proposals] get
+implemented, networking is one scenario that doesn't have a stable API yet,
+which limits current applications compiled to WASI, and restricts the types of
+workloads that can be executed in WASI runtimes.
+
+Today, [we are releasing a library][gh] which adds experimental and _temporary_
```

Comment:
"Today we are releasing" no comma needed there.

"which adds temporary support for outgoing HTTP requests for WASI modules running in Wasmtime. This is an experiment; it's intended to provide a _temporary_ workaround until until the WASI networking API is stable. We expect that once the WASI sockets proposal gets adopted and implemented in language toolchains this library will vanish."

something like that.